### PR TITLE
Improve Char and Entity update packet flow, async-ify a number of db calls, more info for Tracy

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

It would be worth going through each commit, as they're somewhat sane chunks of work. 

Huge thanks to InoUno and Eden for their packet work, it's truly a lightspeed step forward!

### CODE REVIEW

File: scripts/globals/trust.lua [hunk 1]
```diff
@@ -232,6 +232,11 @@ xi.trust.onTradeCipher = function(player, trade, csid, rovCs, arkAngelCs)
 end
 
 xi.trust.canCast = function(caster, spell, not_allowed_trust_ids)
+    -- GMs can do what they want
+    if caster:getGMLevel() > 0 and caster:checkNameFlags(0x04000000) then
```
Comment (TeoTwawki) [inline scripts/globals/trust.lua]: hey, some of us have minimalist GM's we need to have a flag on but don't give access to a lot of powerful commands on. GM commands give access to trusts if they are enabled. if they are disabled NOBODY server wide should be casting. this should be below line 241-243 for this reason.

File: src/map/entities/baseentity.h [hunk 2]
```diff
@@ -158,18 +158,29 @@ enum UPDATETYPE
...
     UPDATE_ALL_MOB  = 0x0F,
     UPDATE_ALL_CHAR = 0x1F,
+    UPDATE_DESPAWN  = 0x20,
 };
 
 enum ENTITYFLAGS
 {
     FLAG_NONE          = 0x000,
     FLAG_INFO_ICON     = 0x001, // (I) Icon next to name
+    FLAG_LARGE_MODEL   = 0x002,
```
Comment (TeoTwawki) [inline src/map/entities/baseentity.h]: no, wrong.

these flags values are
1. not consistent across entity types (that bit isn't size for players), and
2. sizing flags do not have 100% consistent  behaviors across mob models (in some cases, this exact value of 2 will make a model _SMALLER_ so this is a bad way to name the flag)

File: scripts/globals/trust.lua (hunk 1 above)
Comment (zach2good) [reply scripts/globals/trust.lua]: Agreed 👍

File: src/map/entities/baseentity.h (hunk 2 above)
Comment (zach2good) [reply src/map/entities/baseentity.h]: How about `FLAG_ALTERNATE_APPEARANCE` ?

Comment (zach2good) [reply src/map/entities/baseentity.h]: We talked about this on discord, going to leave this out for now

**Review by claywar — APPROVED** [review]

**Pull request merged**